net.pcsx2.PCSX2.json

```diff
@@ -11,11 +11,7 @@
         "--share=ipc",
         "--socket=x11",
         "--socket=pulseaudio",
-        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
-        "--filesystem=xdg-run/dconf",
-        "--filesystem=~/.config/dconf:ro",
-        "--talk-name=ca.desrt.dconf",
-        "--filesystem=host:ro",
+        "--filesystem=host",
```

Comment:
Don't sneak in a permission change...

Reply:
This error occurs on Endless OS

![image](https://user-images.githubusercontent.com/26097419/79698227-f6a80f80-825d-11ea-8c3a-c697cf1dfa1b.png)